python-packages/middlewares/tox.ini

```diff
@@ -0,0 +1,25 @@
+# tox (https://tox.readthedocs.io/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
+[tox]
+envlist = py37
+
+[testenv]
+commands =
+    pip install -e .[dev]
+    python setup.py test
+
+[testenv:run_tests_against_test_deployment]
+commands =
+    # install dependencies from real PyPI
+    pip install eth-account eth-keys eth-utils mypy_extensions web3 pytest
```

Comment:
make sure this matches the list of dev dependencies in setup.py.  Sucks there's this duplication, but that's how it is :/

Tox is not integrated with CircleCI at this time, so please make sure to run it locally to ensure that this file works as expected.

Reply:
https://github.com/0xProject/0x-monorepo/pull/1713/commits/a27c4912759f3f81fc53f2d73bf2a10594098ad8